jbmc/src/java_bytecode/synthetic_methods_map.h

```diff
@@ -27,6 +27,9 @@ enum class synthetic_method_typet
   /// These are generated for both user and stub types, to ensure the actual
   /// static initializer is only run once on any given path.
   STATIC_INITIALIZER_WRAPPER,
+  /// Only exists if the `--static-values` option was used.
+  /// TODO Currently has no content, future commits will add it.
+  JSON_STATIC_INITIALIZER,
```

Comment:
`STATIC_INITIALIZER_VALUES` ?

Reply:
This is now called `USER_SPECIFIED_STATIC_INITIALIZER`, for consistency with the (new) name of the new function symbol.